en/identity-server/next/docs/apis/restapis/rule-metadata.yaml

```diff
@@ -0,0 +1,378 @@
+openapi: 3.0.1
+info:
+  title: WSO2 Identity Server Rule Metadata REST API
+  description: "The Rule Metadata API provides the essential metadata required to configure rules dynamically based on the flow context. \nThe metadata retrieved by this API is designed to support rule configuration in the user interface (UI) by delivering information about each configurable field, the set of applicable comparison operators (e.g., equals, contains) for each field, and lists of possible values or options for fields.\n"
```

Comment:
```suggestion
  description: "The Rule Metadata API provides the essential metadata required to configure rules dynamically based on the authentication flow context. \nThe metadata retrieved by this API is designed to support rule configuration in the user interface (UI) by delivering information about each configurable field, the set of applicable comparison operators (e.g., equals, contains) for each field, and lists of possible values or options for fields.\n"
```

Reply:
This is not just related to the authentication flow. It could be registration, access token flow, profile update flow etc.

Reply:
Ack